### ISSUE TITLE
Language keeps changing even if "Always use English language of the app" is enabled

### ISSUE THREAD

**Issue by floppyD** [Adamantcheese/Kuroba#14]
Looks like it usually happens when you open the captcha, but it sometimes change randomly.

**Comment by Adamantcheese** [comment]
I took a look at it and apparently the current way of doing it is deprecated. So I tried setting it up with the proper method, which would require me to change the base context of the entire application before the super constructor is called (because once it's set you can't change it). But in order for that to happen, the setting for forcing english needs to be checked beforehand, which requires the base context to be set. But it's not set. So the application crashes.

Basically it's a catch-22 here and I don't know how to fix it while using the in-app settings.

I can consider removing translations entirely though, because half of everything in a bunch of locales isn't translated.

**Comment by floppyD** [comment]
I wouldn't mind removing translations to be honest. 4chan is an english website, if you can post on 4chan you should be able to use an app in english.
